### PR TITLE
Add templates of lifelong learning and mutli-edge-inference

### PR DESCRIPTION
**Pull request by JimmyYang20**

### CODE REVIEW

File: docs/proposals/algorithms/lifelong-learning/template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Lifelong learning: Your Algorithm
+
+## Motivation
+### Goals
+
+## Proposal
```
Comment (MooreZheng) [inline docs/proposals/algorithms/lifelong-learning/template.md:6]: Add scope under proposal

File: docs/proposals/algorithms/multi-edge-inference/template.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Multi edge inference: Your Algorithm
+
+## Motivation
+### Goals
+
+## Proposal
```
Comment (MooreZheng) [inline docs/proposals/algorithms/multi-edge-inference/template.md:6]: Add scope under proposal

File: docs/proposals/algorithms/lifelong-learning/template.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+### Use Cases
+
+## Design Details
+### Implementation
+### Customize algorithm
+
+
```
Comment (MooreZheng) [inline docs/proposals/algorithms/lifelong-learning/template.md:13]: Add ##Roadmap

File: docs/proposals/algorithms/multi-edge-inference/template.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+### Use Cases
+
+## Design Details
+### Implementation
+### Customize algorithm
```
Comment (MooreZheng) [inline docs/proposals/algorithms/multi-edge-inference/template.md:11]: Add ## Roadmap

File: docs/proposals/algorithms/lifelong-learning/template.md (hunk 1 above)
Comment (JimmyYang20) [reply docs/proposals/algorithms/lifelong-learning/template.md:6]: added it

File: docs/proposals/algorithms/multi-edge-inference/template.md (hunk 2 above)
Comment (JimmyYang20) [reply docs/proposals/algorithms/multi-edge-inference/template.md:6]: added it

File: docs/proposals/algorithms/lifelong-learning/template.md (hunk 3 above)
Comment (JimmyYang20) [reply docs/proposals/algorithms/lifelong-learning/template.md:15]: added it

File: docs/proposals/algorithms/multi-edge-inference/template.md (hunk 4 above)
Comment (JimmyYang20) [reply docs/proposals/algorithms/multi-edge-inference/template.md]: added it

File: docs/proposals/algorithms/lifelong-learning/template.md (hunk 3 above)
Comment (JimmyYang20) [reply docs/proposals/algorithms/lifelong-learning/template.md:15]: added it

**Pull request merged**